internal/env/env_test.go

```diff
@@ -0,0 +1,43 @@
+package env
+
+import (
+	"github.com/stretchr/testify/require"
+	"os"
+	"testing"
+)
+
+func TestGetEnvBool(t *testing.T) {
+	envVarTestKey := "WAYPOINT_GET_ENV_BOOL_TEST"
+	require := require.New(t)
+
+	t.Run("Unset env var returns default", func(t *testing.T) {
```

Comment:
This is all fine but a table test would probably be more appropriate here.

Reply:
I started with that, but stopped when I realized I needed to set an env var for each table entry. That didn't need to stop me though.